lib/SILOptimizer/SILCombiner/SILCombinerApplyVisitors.cpp

```diff
@@ -1393,6 +1582,12 @@ SILInstruction *SILCombiner::visitApplyInst(ApplyInst *AI) {
   return nullptr;
 }
 
+SILInstruction *SILCombiner::visitBeginApplyInst(BeginApplyInst *BAI) {
+  if (tryOptimizeInoutKeypath(BAI))
```

Comment:
Should we try this more generally on all address values, not only `begin_apply`s?

Reply:
This is the call of the keypath runtime function itself. It can only be an apply or begin_apply.

Reply:
Thanks, sorry, I got it confused with "begin_access".